thunder/core/rematerialization.py

```diff
@@ -162,6 +162,13 @@ def apply_rematerialization_for_consumer(
         filter(lambda x: x.name not in map(lambda x: x.name, new_consumer_args), consumer.args)
     )
 
+    # TODO: better investigate. In the case where there are no tensors to rematerialize it is
```

Comment:
Can you put this TODO for investigation in an issue please? There's no need to have it here because it doesn't add value to understand why an early return is here nor it provides clear actions for potential modification of the code around the comment.